lib/task.js

```diff
@@ -140,8 +140,19 @@ export default class Task {
         this.task.payload, this.status.taskId
     );
 
+    let taskCapabilities = this.task.payload.capabilities || {};
+    let privilegedTask = taskCapabilities.privileged || false;
+    if (privilegedTask && !this.runtime.dockerConfig.allowPrivileged) {
```

Comment:
Looking at existing docker-worker scopes... Maybe `docker-worker:capability:privileged` would be better... Adding `:<provisionerId>/<workerType>` might be nice too..

Anyways, I could imagine adding more `docker-worker:capability:...` style scopes in the future.
Example: `docker-worker:capability:cpu-shares` to allow specifying CPU-shares higher than some allowed constant....

Note, scopes are bad for things continuous resources like say cpu-shares, or memory-limit, but can still be used.. We just hard code some scope covered intervals... like `docker-worker:capability:memory-1g` and `docker-worker:capability:memory-2g`, `docker-worker:capability:memory-10g` , `docker-worker:capability:unlimited-memory`, then when someone asks for "2g" memory we have to check for one of 3 scopes...

Anyways, that's how I plan to solve it for scoping task expiration and artifact sizes when someday we do that...